tests/pkg_cryptoauthlib_internal-tests/Makefile

```diff
@@ -0,0 +1,9 @@
+include ../Makefile.tests_common
+
+INCLUDES += -I$(RIOTBASE)/pkg/cryptoauthlib/include
```

Comment:
Isn't this already set in `pkg/cryptoauthlib/Makefile.include`? (otherwise, it should)